ui/app/styles/core.scss

```diff
@@ -120,5 +120,4 @@
 @import './components/unseal-warning';
 // @import './components/ui-wizard'; // remove, see PR https://github.com/hashicorp/vault/pull/19220
 @import './components/vault-loading';
-@import './components/vlt-radio';
```

Comment:
💜 to see this one gone. Shamir flow was a hold-out for some old styles that I couldn't easily remove.